docs/Building.md

```diff
@@ -37,3 +37,17 @@ $ sbt server/docker
 ```
 
 This will build and tag a Docker image with a name like `broadinstitute/cromwell:<VERSION>-SNAP`.
+
+###IntelliJ gotchas
+
+If you develop Cromwell using IntelliJ IDEA, you may the following error message on startup:
```

Comment:
verb AWOL

Reply:
`4100 MB` is a lot. That much memory should only be required for `sbt clean test …`<sup>1</sup>. I believe the IDE shouldn't need that much for just dumping our complex sbt project. I've been able to get away (so far) with `1536`, which is just double IntelliJ IDEA's default `768`.

---
<sup>1</sup> In addition to the project and compilation using a lot of memory the tests run within the sbt process. Once they're done the tests do not seem to be cleaning up after themselves correctly. When other sbt tasks run after `test` there often isn't enough memory "left", thus `4g`.

Reply:
thx